drivers/SmartThings/matter-switch/profiles/eve-energy.yml

```diff
@@ -0,0 +1,18 @@
+name: eve-energy
```

Comment:
I would just name this profile `power-energy-powerConsumption`, since it's generic enough that other devices may end up using it.

Reply:
Because of some unintended changesets in this PR, I created a new PR at #940

In this new PR, we renamed the profile to `power-energy-powerConsumption`